src/category_theory/limits/cones.lean

```diff
@@ -586,6 +586,54 @@ def map_cocone_inv_map_cocone {F : J ⥤ D} (H : D ⥤ C) [is_equivalence H] (c
   map_cocone_inv H (map_cocone H c) ≅ c :=
 (limits.cocones.functoriality_equivalence F (as_equivalence H)).unit_iso.symm.app c
 
+/--
+`map_cone` commutes with `postcompose`
+-/
+def map_cone_postcompose {α : F ⟶ G} {c} :
```

Comment:
Won't we need simp lemmas for the projections `hom` and `inv` etc?

Reply:
Added!